docs/AppExtensions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: app-extensions
+title: App Extensions
+layout: docs
+category: Guides (iOS)
+permalink: docs/app-extensions.html
+next: native-modules-android
+previous: building-for-apple-tv
+---
+
+Building app extensions with React Native is currently not supported because of memory issues.
```

Comment:
Very true. I figured that making a bold statement like "It doesn't work" would get enough disagreement for it become more refined 😆